TAScheduler/AcceptanceTests/test_logout.py

```diff
@@ -0,0 +1,19 @@
+from django.test import TestCase, Client
+from django.contrib.auth.models import User
+
+class LogoutTest(TestCase):
+    def setUp(self):
+        self.client = Client()
+        self.user = User.objects.create_user(username='testuser', password='12345')
+        self.client.login(username='testuser', password='12345')
+
+    def test_logout(self):
+        # User is logged in here
+        response = self.client.get('/logout/')  # Assuming /logout/ is the URL for the logout view
```

Comment:
The way the prototypes are setup, we would need to start from "home/" to perform a logout.